frontend/app/api/concepts/movie_appearances.json

```diff
@@ -2,7 +2,13 @@
   "action_movies": {
     "parent": "movie_appearances",
     "label": "Action Movies",
-    "matchingEntries": 8234
+    "matchingEntries": 8234,
+    "additionalInfos": [
+      {
+          "key": "## Action film",
```

Comment:
This will be rendered as an h2 -- however we apply some weird styling to h2 elements making them look like the headers of the editor tabs.

What should we do about it? We could limit that styling to the specific parts of the app, or should we discourage the use of markdown headers greater than, say, level-four headers?